_data/sites.json

```diff
@@ -19059,7 +19059,7 @@
         "domains": [
             "which.co.uk"
         ],
-        "notes": "Require your membership number to be sent in the email. This can be found here <a href=\"https://www.which.co.uk/account/#/\">https://www.which.co.uk/account/#/</a>",
+        "notes": "Require your membership number to be sent in the email. This can be found here [https://www.which.co.uk/account/#/](https://www.which.co.uk/account/#/)",
```

Comment:
```suggestion
        "notes": "Require your membership number to be sent in the email. This can be found [here](https://www.which.co.uk/account/#/)",
```